packages/amplication-client/src/VersionControl/PendingChanges.scss

```diff
@@ -85,12 +87,16 @@ $counter-height: 18px;
     padding: var(--default-spacing);
     border-radius: var(--small-border-radius);
     height: 100%;
+    overflow-x: hidden;
+    overflow-y: auto;
```

Comment:
do we need this?

Reply:
Yes, its to block the commit message from by in the scroll
<img width="1512" alt="Screen Shot 2022-08-21 at 19 42 57" src="https://user-images.githubusercontent.com/61761153/185801751-fd977a5c-8e2d-4b2e-9393-13c122d9229b.png">